<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_sys_manage_outer_pay_set_tax_type</class>
 <widget class="QDialog" name="w_sys_manage_outer_pay_set_tax_type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,1">
     <item row="0" column="1">
      <widget class="lds_ComboBox" name="comboBox_type"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
       <item row="0" column="0">
        <widget class="lds_checkbox_wrap" name="checkBox_tax1">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="pushbutton_underline" name="pushButton_change_taxname1">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="lds_checkbox_wrap" name="checkBox_tax2">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="pushbutton_underline" name="pushButton_change_taxname2">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="lds_checkbox_wrap" name="checkBox_tax3">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="pushbutton_underline" name="pushButton_change_taxname3">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>pushbutton_underline</class>
   <extends>QPushButton</extends>
   <header>pushbutton_underline.h</header>
  </customwidget>
  <customwidget>
   <class>lds_checkbox_wrap</class>
   <extends>QCheckBox</extends>
   <header>lds_checkbox_wrap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
